soweego/linker/workflow.py

```diff
@@ -180,17 +180,20 @@ def in_both_datasets(col: str) -> bool:
 
 def init_model(classifier, binarize, number_of_features):
     # TODO expose other useful parameters
-    if classifier is rl.NaiveBayesClassifier:
-        model = classifier(binarize=binarize)
-    elif classifier is rl.SVMClassifier:
-        model = classifier()
-    # FIXME bad hack due to import issues in constants
-    elif classifier is None:
+    if classifier == constants.NAIVE_BAYES_CLASSFIER:
```

Comment:
```suggestion
    if classifier is constants.NAIVE_BAYES_CLASSIFIER:
```
Is there a reason why you compare with `==` and not `is`?